sogura/sogura.md

```diff
@@ -0,0 +1,13 @@
+# FirstName LastName (Unique ID)  Shuta Ogura (sogura)
+## Profile Pic:
```

Comment:
May be you can add your profile pic in the same folder and add markup link here?